docs/development/extensions-core/azure.md

```diff
@@ -40,5 +40,6 @@ To use this Apache Druid extension, [include](../../configuration/extensions.md#
 |`druid.azure.protocol`|the protocol to use|http or https|https|
 |`druid.azure.maxTries`|Number of tries before canceling an Azure operation.| |3|
 |`druid.azure.maxListingLength`|maximum number of input files matching a given prefix to retrieve at a time| |1024|
+|`druid.azure.endpointSuffix`|the endpoint suffix to use.|Examples: `core.windows.net`, `core.usgovcloudapi.net`|`core.windows.net`|
```

Comment:
can you add some more colour here? when would one want to override this? Pointing to external documentation is fine too.

Reply:
added usecase with link to docs